edx_rest_framework_extensions/auth/jwt/middleware.py

```diff
@@ -122,7 +122,7 @@ def _get_missing_cookie_message_and_metric(self, cookie_name):
         request_jwt_cookie = 'missing-{}'.format(cookie_name)
         return cookie_missing_message, request_jwt_cookie
 
-    def process_request(self, request):
+    def process_view(self, request, view_func, view_args, view_kwargs):  # pylint: disable=unused-argument
```

Comment:
@douglashall: Do you see any issue with this particular change?  The `process_request` call was happening for all middleware before `process_view`, so when I tried introducing another middleware that could change the request before this one, but needed to use `process_view`, it was getting called too late.  Updating this to `process_view` allows the other one to set the header.  

See https://github.com/edx/ecommerce/pull/2531/files . I do plan on moving this middleware into this file (and probably PR), so it will eventually be more understandable.

Reply:
@douglashall: Please ignore this comment for now.  This PR will be updated.

Reply:
I'm marking my own question as resolved.